File: Lista2/Exercícios_python/Sala/6.py
a = float(input("Digite o valor de a: "))
b = float(input("Digite o valor de b: "))
c = float(input("Digite o valor de c: "))
 
if a!=0:
    d = (b**2)-(4*a*c)
    if d<0:
        print("A equação não tem solução")
    elif d==0:
        x1 = -b/(2*a)
        x2 = x1
        print("A equação tem duas raízes reais e iguais, x1=", round(x1, 2), "e x2=", round(x2, 2))
    else:
        x1 = (-b+d**0.5)/(2*a)
        x2 = (-b-d**0.5)/(2*a)
        print("A equação tem duas raízes reais e diferentes, x1=", round(x1, 2), "e x2=", round(x2, 2))
else:
    print("Não é uma equação de segundo grau")

File: Listas/Lista2/Exercícios_C/Sala/6.c
#include <stdio.h>
#include <math.h>
int main ()

{
    float a, b, c, d, x1, x2;
    printf("Digite o valor de A: ");
    scanf("%f", &a);
    printf("Digite o valor de B: ");
    scanf("%f", &b);
    printf("Digite o valor de C: ");
    scanf("%f", &c);

    if (a!=0) {
        d = (pow(b, 2))-(4*a*c);
        if (d<0) {
            printf("A equação não tem solução.");
        }
        else if (d==0) {
            x1 = -b/(2*a);
            x2 = x1;
            printf("A equação tem duas raízes reais e iguais\nx1=%.2f\nx2=%.2f", x1, x2);
        }
        else {
            x1 = (-b+pow(d, 0.5))/(2*a);
            x2 = (-b-pow(d, 0.5))/(2*a);
            printf("A equação tem duas raízes reais e diferentes\nx1=%.2f\nx2=%.2f", x1, x2);
        }
    }
    else {
        printf("Não é uma equação de segundo grau.");
    }
}
